src/transformers/models/albert/modeling_albert.py

```diff
@@ -936,7 +937,7 @@ def forward(
         )
 
 
-class AlbertMLMHead(nn.Module):
+class AlbertMLMHead(nn.Module, GenerationMixin):
```

Comment:
Mmmm pretty sure it's not this one but `AlbertForMaskedLM` that needs to inherit!